### ISSUE TITLE
Not working for any Spotify links

### ISSUE THREAD

**Issue by milovincent** [AllToMP3/alltomp3-app#108]
Using Mac app on High Sierra. Downloads don't even start, with Spotify songs showing up as the full link instead of the title

**Comment by milovincent** [comment]
Trying other links, none seem to work

**Comment by dammy95** [comment]
Having the same problem. Not even downloading Youtube links as well! Have you found any alternatives?

**Comment by milovincent** [comment]
Well obviously there's [youtube-dl](https://rg3.github.io/youtube-dl/), which this utility supposedly uses, and which works for a majority of sites (except for Spotify, which is why I'm not using it instead)

**Comment by jtandrew** [comment]
Seeing the same issue on 0.3.8.  Downloaded 0.3.9 and I am unable to download songs or albums from Spotify

**Comment by RosalineAng1** [comment]
Can confirm this issue exists on the windows version too

**Comment by RosalineAng1** [comment]
Related to your issue: Apparently it's not working with Deezer links either it would appear (least on Windows it's not. Not sure if you're having this issue as well)

**Comment by oldfruit** [comment]
Not working on my Mac or PC with Spotify links.

**Comment by EsmailELBoBDev2** [comment]
yup after updating to  0.3.9 i can not download spotify songs on my linux PC

**Comment by sWamPyw** [comment]
I'm using windows 10, alltomp3 version 0.3.9.
the spotify playlist and youtube videos i put in won't download.
The app is not showing any sign of progression even after leaving it open over 30 minutes.

Spotify playlist i tried to download:
https://open.spotify.com/user/de_hunted/playlist/5GaObtkpsi3hBr1yfBOZRh?si=Ld7b36LlTquioClJrrOQyA

**Comment by RosalineAng1** [comment]
version 0.3.10 doesn't fix this issue either....

**Comment by RosalineAng1** [comment]
so version 0.3.10 may of fixed the issue but only seems to work with US song...anything in foreign language it won't work on

**Comment by NTag** [comment]
Could you give me a precise example of a song/URL which doesn't work? And are you on Windows, macOS or Linux?

**Comment by maneeshk5** [comment]
I'm using Mac and Macbook.
i cannot download any song from spotify.
mac is old and macbook is latest one. Song/URL is not working on both.

**Comment by Matt0017** [comment]
I am on a Windows 10 and have tried with https://open.spotify.com/album/2AluSp6sapRu17yc4r2a5F?si=lGwAa8eER-CMZ2DQDmJrtg and didn't work

**Comment by maneeshk5** [comment]
Like this link is not downloading,
and there are more.
https://open.spotify.com/track/6l0Ky2PycJV1Bz88kFeVGL?si=t8PIoJWESPCoeWXmqcI2Pg

**Comment by maneeshk5** [comment]
![image](https://user-images.githubusercontent.com/23071640/57225724-a624ab80-7026-11e9-9620-4624bcdad7f7.png)

**Comment by maneeshk5** [comment]
it is stopped on 50% as you can see

**Comment by dkennard3** [comment]
What fixed the issue for me was that I had to create a folder to store the downloaded songs into. Which makes sense because otherwise the downloads wouldn't have anywhere to go. 
![image](https://user-images.githubusercontent.com/46568557/57239600-5334ff00-6ffa-11e9-8f5c-25dbab3b50e1.png)

I clicked on the little pencil icon after I made a folder on my desktop called "Downloaded songs", once I did that it worked fine for me

**Comment by sWamPyw** [comment]
some Songs are not downloading again. the playlist im trying to download:
https://open.spotify.com/user/de_hunted/playlist/27nWMkGl4yy9HgCrP6OlHI?si=wt3oI7tESZeFEPkzlJ-XKw

**Comment by SajidA829** [comment]
This is kinda late but I found a workaround, try this website to download your songs: https://free-mp3-download.net/

**Comment by egg98** [comment]
> Could you give me a precise example of a song/URL which doesn't work? And are you on Windows, macOS or Linux?

Hi! I'm having the same issue. Nothing downloads :(
![image](https://user-images.githubusercontent.com/34256057/79029873-59b1ec00-7b64-11ea-9b86-ab5b95a9bf4f.png)

**Comment by iarapakis** [comment]
Same here. I tried the MacOS version and attempted to download this public playlist I created, with no success: https://open.spotify.com/playlist/2Ozx22YrHgXIUSW3wgjR8u?si=0E4E3phMQMinFXZVureVsA

Any thoughts as to why it's not working with Spotify and if this can be fixed? The idea behind the app is amazing and it's a pitty that it's not able to deliver what it promises.

**Comment by freewaydog** [comment]
Having the same problem on Windows 10
![Screenshot 2021-11-09 083836](https://user-images.githubusercontent.com/93991360/140934093-20f78047-c0e3-4c73-b4db-3660f072a002.png)
:
